File: src/main/java/it/contrader/dto/RelationshipDTO.java
package it.contrader.dto;

import it.contrader.model.EntityCustomer;
import it.contrader.model.EntityOwner;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class RelationshipDTO {

	private Long id;

	private EntityOwner entityowner;

	private EntityCustomer entitycustomer;


}

File: src/dto/relationshipdto.ts
import { EntityOwnerDTO } from './entityownerdto';
import { EntityCustomerDTO } from './entitycustomerdto';

/**
 * Classe DTO di User. DEVE essere uguale (stesso nome classe, stessi attributi e stessi nomi) a
 * quello nel backend.
 * 
 * @author Salvatore Petrianni
 */
export class RelationshipDTO {

   id: number;

   entityowner: EntityOwnerDTO;

   entitycustomer: EntityCustomerDTO;

}

